<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickLinxClass</class>
 <widget class="QMainWindow" name="QuickLinxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickLinx</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===== Window background ===== */
QMainWindow {
    background-color: #005F86;
}

/* ===== Status label ===== */
QLabel#status_label {
    color: #f1f1f1;
    font-size: 14px;
    font-weight: 500;
}

/* ===== Buttons ===== */
QPushButton {
    background-color: #2c2c30;
    color: #f5f5f5;
    border: 1px solid #3c3c40;
    border-radius: 6px;
    padding: 4px 18px;
}

QPushButton:hover {
    background-color: #3a3a3f;
	cursor: pointer;
}

QPushButton:pressed {
    background-color: #505056;
}

/* Overwrite Button Style*/
QPushButton#overwrite_button {
   	background-color: #5c2a2a;
	border: 1px solid #7a3434;
}

QPushButton#overwrite_button:hover {
    background-color: #733333;
	
}

QPushButton#overwrite_button:disabled {
    background-color: #4a2b2b;   /* muted red */
    border-color: #5a3333;
    color: #888;
}


/*Merge Button Style*/
QPushButton#merge_button {
	background-color: #ebb54f;
	border: 1px solid #b88622;
	color: #000;
}

QPushButton#merge_button:hover {
    background-color: #d9c76e;
}

QPushButton#merge_button:disabled {
    background-color: #4a4425;   /* muted gold */
    border-color: #5a4d2a;
    color: #888;
}


/* ===== Progress bar ===== */
QProgressBar {
    border: 1px solid #3c3c40;
    border-radius: 4px;
    background-color: #26262a;
    color: #f1f1f1;
    text-align: center;
}

QProgressBar::chunk {
    background-color: #F29B2B;   /* accent color */
    border-radius: 4px;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="vertical_layout">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="status_label">
        <property name="text">
         <string>status_label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontal_layout">
        <item>
         <widget class="QPushButton" name="export_button">
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="import_button">
          <property name="text">
           <string>Import</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="merge_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Merge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="overwrite_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Overwrite</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QuickLinx.qrc"/>
 </resources>
 <connections/>
</ui>
